### DISCUSSION TITLE
Coco training logs

### DISCUSSION THREAD

**Discussion by PushpakBhoge512** [ultralytics/yolov5#8926] [Q&A] [answered]
Hello, I was just wondering, are the training loss and the other pr curves, etc basically everything which gets stored in a folder. are all these logs available for the provided pre-trained models?

These curves and logs can be really helpful in understanding how the model evolves throughout the training for such a huge dataset since most people can not afford to train models on such big datasets. This could help to understand questions like how long to train the model. how the loss curves look over longer duration etc.

So, it would be really great if you could share those logs
thank you

**Comment by glenn-jocher** [comment, marked as answer]
@PushpakBhoge512 all v6.1 trainings are logged here:
https://wandb.ai/glenn-jocher/YOLOv5_v61_official

**Reply by PushpakBhoge512** [reply]
Thank you so much for the logs !

**Reply by glenn-jocher** [reply]
You're welcome!
